components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -456,7 +456,11 @@ function AuctionDetails(props: Props) {
                     {auctionDetails && auctionDetails!.enchantments.length > 0 ? (
                         <ul className={styles.list}>
                             {auctionDetails?.enchantments.map(enchantment => {
+                                if(enchantment.name === "Ultimate Reiterate"){
+                                    enchantment.name = "Duplex";
```

Comment:
Is there a specific reason you just called it Duplex? We show all the Ultimate enchants with the "Ultimate" prefix to make it more obvious and easier to find (for example in the item filter search)

Reply:
@Parkusisafk After this is cleared, I will merge the PR :)

Reply:
oops, did ya'll recently changed ultimate enchants to add the prefix "Ultimate"? When I was doing the change I remember that I stumbled across a terminator with just "Soul Eater", so I assumed that the ultimate enchants did not have the "Ultimate" prefix, and instead "Ultimate Reiterate" is just some raw data carried over from the api with the "Ultimate" still on it. Just now I checked and indeed the prefix is present. Thank you for correcting me

To address the ".replace" function, at first I was unsure of the content of the string [enchantment.name], ie. there might be other characters inside (example: Enchantment_Ultimate_Reiterate), and there might be other functions such as .slice that removes the extra part and extract the enchantment. So putting ".replace" was just being cautious